e2e/docker-compose.yml

```diff
@@ -26,14 +26,15 @@ services:
     environment:
       WORDPRESS_DB_HOST: db
       WORDPRESS_DB_NAME: wordpress
-      WORDPRESS_DB_USER: root
+      WORDPRESS_DB_USER: test
```

Comment:
Hey, why did you have to change the WP database user? The e2e tests are failing because of that:

https://github.com/frontity/frontity/pull/755/checks?check_run_id=2135130303#step:5:1444

Reply:
If it's better to use a `test` user, I guess we would have to specify it somewhere else in order to make the test work.